autopush/router/apnsrouter.py

```diff
@@ -163,7 +163,10 @@ def _route(self, notification, router_data):
             elif isinstance(e, (HTTP20Error, socket.error)):
                 reason = "http2_error"
             else:
-                reason = e.extra.get("reason", "unknown")
+                try:
+                    reason = e.extra.get("reason", "unknown")
+                except AttributeError:
+                    reason = format("Unknown: {}", e)
```

Comment:
`reason` here is a metrics tag value which shouldn't have whitespace:
```suggestion
                    reason = "unknown"
```

Sentry's complaining about a `hyper.common.exceptions.ConnectionResetError`,  could include it alongside `ConnectionError` handling above -- `if isinstance(e, (ConnectionError, ConnectionResetError)):` but probably not even worth the effort.